### ISSUE TITLE
Parsing the response leads to an error

### ISSUE THREAD

**Issue by korymath** [gunthercox/django_chatterbot#4]
TypeError: In order to allow non-dict objects to be serialized set the safe parameter to False 

Setting the parameter to False makes the output not display.

**Comment by korymath** [comment]
Fixed with this kind of wrapping in the api.py in the example program. 

```
        data = {
            'text': response_data
        }

        return JsonResponse(data)
```

**Comment by gunthercox** [comment]
This was an issue with the chatterbot module where the incorrect data was being returned from an IO adapter. This has been corrected in the 0.3.6 release of chatterbot (https://github.com/gunthercox/ChatterBot/commit/f99e821a45548f30124ea9c8a82c0bdeed8cc796).

I have added [unit tests](https://github.com/gunthercox/django_chatterbot/tree/master/tests) to the django_chatterbot repo to ensure that this is in fact fixed.
